tests/RobotFramework/tests/cumulocity/shell/c8y_Command

```diff
@@ -1,4 +1,4 @@
 [exec]
-topic = "c8y/s/ds"
+topic ="${.bridge.topic_prefix}/s/ds" 
```

Comment:
> Why is this change needed?

The tests were failing because of the change we implemented with "get_smartrest_custom_operation_topics()" but it is not exactly the fault of that function but rather we were matching the topic wrongly. This tests are using custom prefix `c8y-custom` but operation file used `c8y` prefix as it can be seen in your comment above. So this file was telling us that message will come on topic `c8y/s/ds` but in reality it was `c8y-custom/s/ds`. Previously it was passing because we just checking incoming topic (if it has mapper prefix or not) and not what is actually in the file.